test/spec/modules/medianetBidAdapter_spec.js

```diff
@@ -416,6 +500,31 @@ describe('Media.net bid adapter', () => {
   });
 
   describe('buildRequests', () => {
+    let sandbox;
+    let mock;
+    before(() => {
+      sandbox = sinon.sandbox.create();
+      mock = sinon.mock(window);
+      window.innerWidth = 1000;
+      window.innerHeight = 1000;
```

Comment:
A mock [does not modify the given object](http://sinonjs.org/releases/v1.17.6/mocks/#var-mock--sinonmockobj).
> Does not change the object, but returns a mock object to set expectations on the object’s methods.

This means that setting these read-only values on `window` will most likely throw errors (in Safari I believe) and also not be reset after your test suite runs.  To solve this you can either use a helper function to get the screen dimensions and stub _that_ to get the results you want or you can make your tests more tolerant of various screen sizes (as the browserstack tests will run across multiple devices).